woocommerce/woocommerce
# Missing CSS class on the empty-orders notice CTA in the My Account > Orders page

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.


### Changes proposed in this Pull Request:



This PR adds the missing `wc-forward` class to the empty-orders notice CTA button.

Closes #41324

#### Screenshots

|Before|After|
|---|---|
|![y9ZWJI.png](https://github.com/woocommerce/woocommerce/assets/1105590/96a85e94-01b5-48b8-b55f-373b3c55a831)|![IiNo2t.png](https://github.com/woocommerce/woocommerce/assets/1105590/d8698069-fef4-45cc-9211-1908be9d5b5d)|



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Use a block theme i.e. TT4
2. Ensure that the store uses the cart and checkout blocks and that the store has no orders.
3. Navigate to My Account > Orders
4. Notice the issue.
5. Apply this patch.
6. Ensure that the CTA looks as expected.



### Changelog entry




#### Significance




#### Type




#### Message 
Added missing CSS class on the empty-orders notice CTA in the My Account > Orders page

#### Comment 

</details>
